fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindRelation.java

```diff
@@ -520,4 +531,29 @@ private List<Long> getPartitionIds(TableIf t, UnboundRelation unboundRelation, L
             return part.getId();
         }).collect(ImmutableList.toImmutableList());
     }
+
+    /**
+     * Convert view definition SQL based on current SQL dialect
+     */
+    private String convertViewDefinition(String originSql, ConnectContext ctx) {
```

Comment:
I suggest the extract a common method to do the conversion.
This code is just same as in `convertOriginStmt()` in `ConnectProcessor.java`